ls1intum/Artemis
# `Development`: Fix inconsistent field name in conversation API

### Checklist
#### General



### Motivation and Context


This bug was noticed by @TimOrtel  on production. In `ConversationWebsocketDTO` class `metisCrudAction` field was changed from `crudAction` to `metisCrudAction` in server but not in client side and therefore this caused to not receive the updated conversation through websocket. 

### Description


Renamed the field to `metisCrudAction` on client side.

### Steps for Testing

Prerequisites:
- 2 Users registered at the same course
- 1 Course

1. Log in to Artemis
2. Navigate to messages tab and start chat with 2 users
3. Create new conversation (one-to-one or group chat or channels) and verify that its displayed on the other user (without reloading)
4. Send new message and verify that the unread count badge is displayed on the other user side (in order for badge to be shown the conversation should not be active so do not click the conversation)
5. Navigate to course overview from one of the users and send message from the other user. On **Messages** tab a red circle should be displayed

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test
